config.json

```diff
@@ -507,6 +508,14 @@
       "difficulty": 1,
       "topics": [
       ]
+    },
+    {
+      "slug": "proverb",
+      "difficulty": 1,
+      "topics": [
+        "Array manipulation",
+        "arguments keyword"
```

Comment:
I think this topic is too JavaScript specific. I would use some of the topics listed from the previous comment.